source/jormungandr/jormungandr/planner.py

```diff
@@ -55,6 +56,28 @@ def __init__(
         self.timeframe = timeframe
         self.depth = depth
         self.isochrone_center = isochrone_center
+        self.sn_params = sn_params
+
+
+# Needed for GraphicalIsochrones
+class StreetNetworkParameters(object):
```

Comment:
if you're interested, you can have look at dataclass (available on python3.7), this will allow you to save a lot of codes